module/dice/dhRoll.mjs

```diff
@@ -237,6 +237,40 @@ export default class DHRoll extends Roll {
     }
 }
 
+async function automateHopeFear(config) {
+    const automationSettings = game.settings.get(CONFIG.DH.id, CONFIG.DH.SETTINGS.gameSettings.Automation);
+    const hopeFearAutomation = automationSettings.hopeFear;
+    if (!config.source?.actor ||
+        (game.user.isGM ? !hopeFearAutomation.gm : !hopeFearAutomation.players) ||
+        config.actionType === 'reaction' ||
+        config.tagTeamSelected ||
+        config.skips?.resources)
+        return;
+    const actor = await fromUuid(config.source.actor);
+    let updates = [];
+    if (!actor) return;
+    if (config.roll.isCritical || config.roll.result.duality === 1)
+        updates.push({ key: 'hope', value: 1, total: -1, enabled: true });
+    if (config.roll.isCritical) updates.push({ key: 'stress', value: -1, total: 1, enabled: true });
```

Comment:
This is incorrect. Stress is an `inverted` resource, so it's supposed to be value=1, total=-1.
This change makes stress increase on critical :tongue: